packages/flutter/lib/src/semantics/semantics_event.dart

```diff
@@ -8,6 +8,18 @@ import 'package:flutter/painting.dart';
 
 export 'dart:ui' show TextDirection;
 
+/// Determines the politeness setting of aria live in Flutter web.
+///
+/// It is used to set the priority with which screen reader should treat updates to live regions.
+enum AriaLivePolitenessSetting {
+  /// The screen reader will speak changes whenever the user is idle.
+  polite,
+
+  /// It will interrupt any announcement a screen reader is currently making to notify
+  /// the user about the change. It should only be used for time-sensitive/critical notifications.
```

Comment:
nit: move the second sentence into its own paragraph: https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph